content/overview/overview/what-is-cloudgov.md

```diff
@@ -2,29 +2,31 @@
 menu:
   overview:
     parent: overview
-title: What is cloud.gov?
+title: What Is cloud.gov?
 weight: 10
 aliases:
   - /docs/intro/overview/what-is-cloudgov
   - /intro/overview/what-is-cloudgov
 ---
 
-cloud.gov gives teams working for federal government a secure, fully compliant foundation, integrated with private sector infrastructure providers, on which to build and release products and updates quickly.
+## What is cloud.gov?
```

Comment:
We don't need this extra heading here, since the page will automatically have the title on top ("What is cloud.gov?").